packages/react-proptypes/src/index.ts

```diff
@@ -1,8 +1,9 @@
 import * as _ from 'lodash'
 import * as PropTypes from 'prop-types'
+
 import leven from './leven'
-import { ObjectOf } from '../types'
 
+type ObjectOf<T> = { [key: string]: T }
```

Comment:
nit: `Record<string, T>`? :) but please, feel free to reason by yourself if that should be changed

Reply:
👍